.github/workflows/main.yml

```
@@ -94,6 +94,10 @@ jobs:
           activate-environment: constructor-dev
           environment-file: dev/environment.yml
           python-version: ${{ matrix.python-version }}
+      - name: Install AzureSignTool
+        if: ${{ matrix.os == 'windows' }}
```

```suggestion
        if: matrix.os == 'windows'
```
These can be omitted in `if` clauses, I think.